app/src/main/res/values/strings.xml

```diff
@@ -168,10 +168,10 @@
     <string name="request_timeout">Request Timeout</string>
     <string name="superuser_notification">Superuser Notification</string>
     <string name="settings_su_reauth_title">Reauthenticate after upgrade</string>
-    <string name="settings_su_reauth_summary">Reauthenticate superuser permissions after an application upgrades</string>
-    <string name="settings_su_tapjack_title">Enable Tapjacking Protection</string>
```

Comment:
This is a setting, so "enable" is implied.

Reply:
Right, but it's fine.

Reply:
Nothing is ever fine. It is either sub-optimal or OK.

Reply:
I think it's OK then. Some apps do crazy things where enabling toggles disable features, Magisk is more sensible than that but it's still good to know the feature will explicitly be enabled.

Reply:
Rather than introducing inconsistent conventions in the app, I think letting crazy be crazy is better.
Having positive toggles without the fluff is what Android AOSP does.
This avoids the idea that it could be "on", but disabled.
Also there will be no translations in the imperative mood, which could result in ~do turn on Tapjacking Protection now.